packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -59,6 +59,30 @@ export NewMetadataBackup from './metadata'
 
 // ===================================================================
 
+export const extractSettingsToDisplay = ({
```

Comment:
Either change the name or add a comment to explain the logic behind it, eg:  `returns a map of settings with non-default values`.